style.css

```diff
@@ -0,0 +1,390 @@
+@font-face {
+  font-family: NexaLight;
+  src: url('./Font/NexaLight.otf');
+}
+
+@font-face {
+  font-family: NexaBlack;
+  src: url('./Font/NexaBlack.otf');
+}
+
+@font-face {
+  font-family: NexaBold;
+  src: url('./Font/NexaBold.otf');
+}
+
+* {
+  margin-top: 10px;
```

Comment:
why use this in here * we always works like 

pading:0;
margin:0;

@aeshapatel025

Reply:
chnages done